src/core/server/saved_objects/import/resolve_import_errors.ts

```diff
@@ -150,6 +152,24 @@ export async function resolveSavedObjectsImportErrors({
   };
   const checkConflictsResult = await checkConflicts(checkConflictsParams);
   errorAccumulator = [...errorAccumulator, ...checkConflictsResult.errors];
+  importStateMap = new Map([...importStateMap, ...checkConflictsResult.importStateMap]);
+
+  let originConflictsImportStateMap: ImportStateMap = new Map();
```

Comment:
NIT: unnecessary instantiation, you can spread `undefined`.

```ts
let originConflictsImportStateMap: ImportStateMap;
```

Reply:
I was curious to try this, but it gives me this TS error further down:

![image](https://user-images.githubusercontent.com/5295965/153624358-a8b3fe76-e85f-455d-9811-a700b28b53c6.png)

and if I try to use this:

```ts
let originConflictsImportStateMap: ImportStateMap | undefined;
```

I get this error:

![image](https://user-images.githubusercontent.com/5295965/153624982-cc506f4e-611e-4080-b602-b79a43fa0ba3.png)

I'll leave it as-is 😄